Custom attributes value's type should be a string (#285)
configcat/docs: website/docs/sdk-reference/js-ssr.md

let userObject = new configcat.User('john@example.com');
```

| Parameters   | Description                                                                                                                                                          |
| ------------ | -------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
| `identifier` | **REQUIRED.** Unique identifier of a user in your application. Can be any `string` value, even an email address.                                                     |
| `email`      | Optional parameter for easier targeting rule definitions.                                                                                                            |
| `country`    | Optional parameter for easier targeting rule definitions.                                                                                                            |

> | `custom`     | Optional dictionary for custom attributes of a user for advanced targeting rule definitions. e.g. User role, Subscription type. The value's type of the dictionary must be a `string`. |

Reviewer: ```suggestion
| `custom`     | Optional `dictionary  of strings` representing the custom attributes of a user for advanced targeting rule definitions. e.g. User role, Subscription type. |
```
